src/js/actions.js

```diff
@@ -148,7 +141,9 @@ export function submitForm(formConfig, form) {
         ? formConfig.transformForSubmit(formConfig, form)
         : transformForSubmit(formConfig, form);
 
-      promise = submitToUrl(body, formConfig.submitUrl, recordEvent);
+      promise = formConfig.submitToUrl
```

Comment:
I've added a `formConfig.submitToUrl` option that needs to be documented. The default just removes empty objects and arrays (plus uploads which may possibly go).

Reply:
So now there is  the existing `formConfig.submit` which lets the form creator completely replace the submit process incuding any pre-form massaging, plus the new `formConfig.submitToUrl` for just the network part.

Reply:
So if I'm understanding this correctly, there are now 3 options in the config for submit actions:
1. `submit`: a function to handle the form submission
2. `submitToUrl`: a function to handle the specific part of the submission where data is actually posted, and to log responses back
3. `submitUrl`: a string of the url the data is being submitted to

Is it actually necessary to have `submitToUrl` (2) as an option? It seems like `submitToUrl()` is just a subset of the general `submitForm()` function, and some of the logic in `submitForm()` is actually dependent on logic in `submitToUrl()`, with the checks for the errors. 

Are these functions actually wholly distinct from one another? Can we imagine a world where someone wants to use the default option for one, but write a custom option for the other?  If someone wanted to rewrite both, I think all we would need is the `submit` option in the config, and someone would have to pass a function to that that covers both `submitForm()` and `submitToUrl()` functionality. What do you think?

Reply:
I'd be fine with just having `formConfig.submit` be the only option and exposing these to either be used as-is or as templates for building custom code.

Reply:
I think that might make things simpler!